### ISSUE TITLE
Quick restart all jobs/cluster state

### ISSUE THREAD

**Issue by mariusz-jachimowicz-83** [onyx-platform/onyx#727]
Make restart cluster state easier. 
Based on slack conversation 18.01.2017

rc1140 [3:30 PM] 
hi all is there a way to clear out a cluster without having to rm zookeeper and resetup everything , while testing have a bunch of jobs that are stuck and constantly try to run and are unlikely to ever complete
i am aware of the kill-job api , but im not sure how to get a list of running jobs to pass to that function

gardnervickers [3:35 PM] 
If you want a quick way to start fresh in zookeeper, you can change the tenancy-id on your peers. Everything in Onyx is namespaced by the tenancy-id, so starting your peers with a fresh one will give them a clean slate in ZK.

jasonbell [3:36 PM] 
I always have a `zookeeper-shell` open and rmr /onyx as required

gardnervickers [3:37 PM] 
That may or may not be viable for your situation however, as it will leave the old jobs in ZK, just under a different namespace. Those may need to be cleaned up periodically if you run out of disk space.

[3:39]  
Another option is the Replica Query tools in lib Onyx. https://github.com/onyx-platform/lib-onyx/blob/master/README.md#replica-queries
GitHub
onyx-platform/lib-onyx
lib-onyx - A library to support additional functionality in Onyx.
 
rc1140 [3:40 PM] 
lol i tried ot rmr my /onyx it just dies after a while

[3:40]  
dont have the error on hand

[3:41]  
but it seems to timeout

[3:41]  
but will try the new tenancy-id first

gardnervickers [3:41 PM] 
@rc1140: you will need to restart your peers if you rm /onyx

rc1140 [3:41 PM] 
they should be stopped but will confirm

**Comment by MichaelDrogalis** [comment]
Notably this shouldn't be a restart-oriented process, but a hard kill on all jobs. No jobs, included those that are scheduled and waiting, should be eligible for execution after the application of this log entry.

**Comment by mariusz-jachimowicz-83** [comment]
So you generally want to `kill tenancy` - unreversable process.

**Comment by MichaelDrogalis** [comment]
@mariusz-jachimowicz-83 Right. We don't support job restarts or resumption of of state in 0.9, so anything other than `kill-all-jobs` would be more than I would want to see this with patch.
